datafusion/common/src/utils/mod.rs

```diff
@@ -769,20 +769,6 @@ pub fn set_difference<T: Borrow<usize>, S: Borrow<usize>>(
         .collect()
 }
 
-/// Checks whether the given index sequence is monotonically non-decreasing.
-pub fn is_sorted<T: Borrow<usize>>(sequence: impl IntoIterator<Item = T>) -> bool {
```

Comment:
@alamb @buraksenn Why are we removing a public API method without deprecating it first?

https://docs.rs/datafusion-common/latest/datafusion_common/utils/fn.is_sorted.html

Reply:
Sorry -- that was a mistake. I can make a PR to put it back in w/ deprecated

Reply:
Thanks @alamb

Reply:
Here is PR
- https://github.com/apache/datafusion/pull/14388